SwedbankPaySDK/Classes/Merchant Backend/RootLink.swift

```diff
@@ -13,11 +13,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// `ServerProblemType` URLs
-enum ServerProblemType: String {
-    case InternalServerError = "about:blank" // 500
-    case BadGateway = "https://api.payex.com/psp/errordetail/mobilesdk/badgateway" // 502
-    case GatewayTimeOut = "https://api.payex.com/psp/errordetail/mobilesdk/gatewaytimeout" // 504
-    case SystemError = "https://api.payex.com/psp/errordetail/mobilesdk/systemerror"
-    case ConfigurationError = "https://api.payex.com/psp/errordetail/mobilesdk/configurationerror"
+import Foundation
+
+struct RootLink: Link {
+    let href: URL
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)